README.md

```diff
@@ -619,20 +620,22 @@ spec:
   severity: string # required
   condition: # required
     kind: string
+    op: enum
     threshold: number
     lookbackWindow: duration-shorthand
     alertAfter: duration-shorthand
 ```
 
 ##### Notes (AlertCondition)
 
-- **description** *string*, optional description about the alert contdition, contains at most 1050 characters
+- **description** *string*, optional description about the alert condition, contains at most 1050 characters
 - **severity** *string*, required field describing the severity level of the alert (ex. "sev1", "page", etc.)
 - **condition**, required field. Defines the conditions of the alert
   - **kind** *enum(burnrate)* the kind of alerting condition thats checked, defaults to `burnrate`
-
+  
```

Comment:
Is this a new line or a space as well?

It appears the markdown checker is seeing a space on the line, which causes a failure. Could you check the file and run the markdown checker locally? Details on running the check can be found [here](https://github.com/OpenSLO/OpenSLO/blob/main/CONTRIBUTING.md#making-a-pull-request)

Reply:
I'll take a look at this and test it locally.

Reply:
@kenfinnigan - This time I should have all of the whitespace issues fixed.  That's the output from running it locally.  Let me know if there are any issues this time.  I'll work thru them all.

`C:\Users\Wayne\git\OpenSLO>make run/checks/spell-and-markdown     
yarn check-trailing-whitespaces
yarn run v1.22.19
$ node .github/ci/check-trailing-whitespaces.js
Done in 0.11s.
yarn check-word-lists
yarn run v1.22.19
$ node .github/ci/check-word-lists.js
Done in 0.09s.
yarn cspell --no-progress '**/**'
yarn run v1.22.19
$ C:\Users\Wayne\git\OpenSLO\node_modules\.bin\cspell --no-progress **/**
CSpell: Files checked: 22, Issues found: 0 in 0 files
Done in 1.06s.
yarn markdownlint --ignore 'node_modules/' '**/*.md'
yarn run v1.22.19
$ C:\Users\Wayne\git\OpenSLO\node_modules\.bin\markdownlint --ignore node_modules/ **/*.md
Done in 0.39s.`